newswires/app/conf/SearchPresets.scala

```diff
@@ -139,6 +137,18 @@ object SearchPresets {
     )
   )
 
+  private val PaBusiness = List(
+    SearchParams(
+      text = None,
+      suppliersIncl = List("PA"),
+      categoryCodesIncl = List(
+        "paCat:FFF",
+        "paCat:GXX"
+      ),
+      categoryCodesExcl = List("MCC:SPO")
```

Comment:
Is this the right category code for PA sport? I can only find Reuters stories with this code.